tools/README.md

```diff
@@ -50,6 +50,14 @@ Dev containers are supported in [multiple editors and IDEs](https://containers.d
 
 GUI apps like `ui` or `cabana` can also run inside the container by leveraging X11 forwarding. To make use of it on macOS, additional configuration steps must be taken. Follow [these](https://gist.github.com/sorny/969fe55d85c9b0035b0109a31cbcb088) steps to setup X11 forwarding on macOS.
 
+#### VirtualGL on macOS
+
+Some GUI tools require OpenGL to work properly. To use them with XQuartz on mac, VirtualGL needs to be installed.
```

Comment:
can we just print this as a MOTD style thing when the devcontainer starts up if we detect it's not setup?

Reply:
sure